cos-fleet-catalog-connectors-it/nosql/elasticsearch-it/src/test/groovy/org/bf2/cos/connector/camel/it/elasticsearch/ElasticsearchConnectorIT.groovy

```diff
@@ -0,0 +1,102 @@
+package org.bf2.cos.connector.camel.it.elasticsearch
+
+import groovy.util.logging.Slf4j
+import org.bf2.cos.connector.camel.it.support.KafkaConnectorSpec
+import org.bf2.cos.connector.camel.it.support.TestUtils
+import org.elasticsearch.client.Request
+
+import java.util.concurrent.TimeUnit
+
+import static org.bf2.cos.connector.camel.it.elasticsearch.ElasticsearchSupport.client
+import static org.bf2.cos.connector.camel.it.elasticsearch.ElasticsearchSupport.elasticsearchContainer
+
+@Slf4j
+class ElasticsearchConnectorIT extends KafkaConnectorSpec {
+    private static final String ELASTIC_SECURED_USER = 'elastic'
+    private static final String ELASTIC_SECURED_PASSWORD = 'supersecret'
+    private static final String ELASTIC_SECURED_ALIAS = 'tc-elastic-secured'
+    private static final String ELASTIC_ALIAS = 'tc-elastic'
+
+    def "elasticsearch sink"(String alias, String user, String password) {
+        setup:
+            def elastic = elasticsearchContainer(network, alias, user, password)
+            elastic.start()
+
+            def topic = topic()
+            def payload = """{ "kafka_topic": "${topic}" }"""
+            def client = client(elastic, user, password)
+
+            def get = new Request("GET", "/${topic}")
+            def delete = new Request("DELETE", "/${topic}")
+            def search = new Request("GET", "/${topic}/_search")
+
+            def props = [
+                'kafka_topic' : topic,
+                'kafka_bootstrap_servers': kafka.outsideBootstrapServers,
+                'kafka_consumer_group': UUID.randomUUID().toString(),
+                'elasticsearch_host_addresses': alias,
+                'elasticsearch_cluster_name': topic,
+                'elasticsearch_enable_s_s_l': 'false'
+            ]
+
+            if (user != null && password != null) {
+                props['elasticsearch_user'] = user
+                props['elasticsearch_password'] = password
+            }
+
+            def cnt = connectorContainer('elasticsearch_sink_0.1.json', props)
+
+            cnt.withUserProperty('quarkus.log.category."org.apache.camel.component.elasticsearch".level', 'DEBUG')
```

Comment:
We can now use: https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/blob/8404805044bad4b1019193c684330562f798b888/cos-fleet-catalog-connectors-it-support/src/main/java/org/bf2/cos/connector/camel/it/support/ConnectorContainer.java#L184-L187